### PR TITLE
configurable normalizations

### PR DESCRIPTION
**Pull request by edyoshikun**
This PR adds configurable normalizations for both the source and targets similar to the transformations/augmentations that are applied to the samples.

### CODE REVIEW

File: examples/configs/fit_example.yml [hunk 1]
```diff
@@ -36,6 +36,19 @@ data:
   batch_size: 32
   num_workers: 16
   yx_patch_size: [256, 256]
+  normalizations:
+  - class_path: viscy.transforms.NormalizeSampled
```
Comment (ziw-liu) [inline examples/configs/fit_example.yml:40]: Need indentation here.

File: viscy/utils/meta_utils.py [hunk 2]
```diff
@@ -127,6 +127,7 @@ def generate_normalization_metadata(
                 process=f"calculating channel statistics {channel}/{list(channel_ids)}",
             )
             position_statistics = dataset_statistics | {
+                "dataset_statistics": dataset_level_statistics,
```
Comment (ziw-liu) [inline viscy/utils/meta_utils.py:130]: It should already exist. Can you test with a toy dataset without this line?

https://github.com/mehta-lab/VisCy/blob/529340db3d2750a0a5b2eb04a2a452df32e3d56f/viscy/utils/meta_utils.py#L112-L114

Comment (ziw-liu) [reply viscy/utils/meta_utils.py:130]: I suspect that the dataset we looked at earlier was processed with an older version.

Comment (ziw-liu) [reply viscy/utils/meta_utils.py]: See https://github.com/mehta-lab/VisCy/pull/68/commits/0a15f311453565fa6640c1e3deaa6d7ceadeb008 and https://github.com/mehta-lab/VisCy/pull/68/commits/c13400c0c951e9130ab70e9a255ff97f98676e68

**Comment by edyoshikun** [comment]
@ziw-liu just tested with new dataset using the `preprocessing` and we can do the normalizations

**Review by ziw-liu — APPROVED** [review]

**Pull request merged**